pytest-selenium/regions/toc.py

```diff
@@ -1,14 +1,22 @@
 from selenium.webdriver.common.by import By
-from selenium.webdriver.common.keys import Keys
 
-from pages import content
-from pages.base import Page
 from regions.base import Region
 
 
 class TableOfContents(Region):
     _root_locator = (By.CSS_SELECTOR, "ol")
     _section_link_locator = (By.CSS_SELECTOR, "ol li a")
+    _active_section_locator = (By.CSS_SELECTOR, "[aria-label='Current Page']")
+    _default_page_locator = (By.CSS_SELECTOR, "li:nth-child(2) li:nth-of-type(1) a")
```

Comment:
this logic will not always be right for selecting the default page, and i would prefer to parameterize the default page. i have a thing where i try to avoid using the thing i'm testing to generate my assertions for the thing i'm testing.

@m1yag1 what do you think of this?

Reply:
yep, i agree.

On a separate note, the page object is meant to help expose information for the test. If this logic isn't always going to work then I wonder why that is the case? We do want the page to operate as close as possible to the actual page. It might not be the best to generate test data from but it is our interface w/ the page and useful for other things.

Reply:
in this case the default page just might not always be the first anchor in the second `li`, it depends on the structure of the book

Reply:
@TomWoodward Agree....This css works perfect for books without units. For books with units it gives multiple results but still the first hit would be the default page. So I used this instead of using xpaths with AND/OR condition

Reply:
@Malar-Natarajan can we parameterize it so that we remove the dependency on the book structure?